File: vortexdbg-android/src/main/java/com/vortexdbg/linux/android/dvm/api/ApplicationInfo.java
package com.vortexdbg.linux.android.dvm.api;

import com.vortexdbg.linux.android.dvm.DvmObject;
import com.vortexdbg.linux.android.dvm.VM;

public class ApplicationInfo extends DvmObject<Object> {

    public ApplicationInfo(VM vm) {
        super(vm.resolveClass("android/content/pm/ApplicationInfo"), null);
    }

}

File: vortexdbg-android/src/main/kotlin/com/vortexdbg/linux/android/dvm/api/ApplicationInfo.kt
package com.vortexdbg.linux.android.dvm.api

import com.vortexdbg.linux.android.dvm.DvmObject
import com.vortexdbg.linux.android.dvm.VM

open class ApplicationInfo(vm: VM) : DvmObject<Any?>(vm.resolveClass("android/content/pm/ApplicationInfo"), null)
